platform/qt/src/utils/conversion.hpp

```diff
@@ -31,13 +31,9 @@ class ConversionTraits<QVariant> {
 #endif
     }
 
-    static std::size_t arrayLength(const QVariant& value) {
-        return value.toList().size();
-    }
+    static std::size_t arrayLength(const QVariant& value) { return value.toList().size(); }
 
-    static QVariant arrayMember(const QVariant& value, std::size_t i) {
-        return value.toList()[static_cast<int>(i)];
-    }
+    static QVariant arrayMember(const QVariant& value, std::size_t i) { return value.toList()[static_cast<int>(i)]; }
```

Comment:
Let's ditch `AllowShortFunctionsOnASingleLine`?

With the 120 Column Limit this leads to pretty unreadable one liners.